release-notes/4.7.27.md

```diff
@@ -0,0 +1,436 @@
+  only be chosen by administrators.
+
+- **[CRM-20852](https://issues.civicrm.org/jira/browse/CRM-20852) Show tax term
+  in UI (eg. VAT)
+  ([10640](https://github.com/civicrm/civicrm-core/pull/10640))**
+
+  The term describing the tax (e.g. "VAT", "HST") is now displayed in
+  contribution details when tax is applied.
+
+- **[CRM-20889](https://issues.civicrm.org/jira/browse/CRM-20889) Toggle
+  check_number field on backoffice form as payment form fields
+  ([10680](https://github.com/civicrm/civicrm-core/pull/10680))**
+
+  Check number now appears as a detail of the payment when paying by check rather than an attribute
+  of the contribution as a whole.
+
+- **[CRM-21106](https://issues.civicrm.org/jira/browse/CRM-21106) Move financial
+  type ACL clause for reports into extension
+  ([10904](https://github.com/civicrm/civicrm-core/pull/10904)) (preliminary
+  work)**
+
+  A system check and upgrade message now alert users who use financial type ACLs
+  but have not yet installed the "Access Control by Financial Type for Reports"
+  extension.  These features will soon be removed from core CiviCRM.
+
+### CiviEvent
+
+- **[CRM-21160](https://issues.civicrm.org/jira/browse/CRM-21160) Make
+  event_type_id available in event message templates.
+  ([10977](https://github.com/civicrm/civicrm-core/pull/10977))**
+
+  Event message templates now can include `{$event_type_id}` to display the ID
+  of the event type.
+
+### CiviMember
+
+- **[CRM-21236](https://issues.civicrm.org/jira/browse/CRM-21236) Make contact
+  custom fields available in Membership Detail report
+  ([11042](https://github.com/civicrm/civicrm-core/pull/11042))**
+
+  Custom fields about contacts can now be included as columns and filters on the
+  Membership Detail report.
+
+### WordPress Integration
+
+- **[CRM-17633](https://issues.civicrm.org/jira/browse/CRM-17633) WordPress in
+  own directory breaks CiviCRM
+  ([11031](https://github.com/civicrm/civicrm-core/pull/11031) and
+  [105](https://github.com/civicrm/civicrm-wordpress/pull/105))**
+
+  CiviCRM now supports a non-default but acceptable WordPress configuration
+  where WordPress files are in a subdirectory of the site root.
+
+- **[CRM-21243](https://issues.civicrm.org/jira/browse/CRM-21243) Logo in WP
+  menu doesn't follow standard
+  ([118](https://github.com/civicrm/civicrm-wordpress/pull/118))**
+
+  The CiviCRM icon in the WordPress left-side menu now is a SVG that adopts the
+  appropriate colors as the menu item is highlighted or selected.
+
+## <a name="bugs"></a>Bugs resolved
+
+### Core CiviCRM
+
+- **[CRM-21229](https://issues.civicrm.org/jira/browse/CRM-21229) Manage Group
+  page is slow if you have smart groups
+  ([11033](https://github.com/civicrm/civicrm-core/pull/11033)) (partially
+  resolved)**
+
+  The Manage Groups page now will display "Unknown" for the count of smart group
+  members where the smart group cache is empty.  You can click the button to
```

Comment:
@agh1 actually the "Unkown" is set for the count when the smart group cache got outdated. Can you change ```empty``` to ```outdated``` here?

Reply:
Absolutely--thanks for catching this distinction!